<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmittMeasMainWindow</class>
 <widget class="QMainWindow" name="EmittMeasMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emittance Measurement</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>

    <!-- Optics -->
    <item>
     <widget class="QGroupBox" name="groupBoxDataOptics">
      <property name="title">
       <string>Data loading</string>
      </property>
      <layout class="QGridLayout" name="gridLayoutDataOptics">

       <!-- Load Twiss file -->
       <item row="1" column="0">
        <widget class="QLabel" name="labelTwissFile">
         <property name="text">
          <string>Load data file:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="twissFileLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="loadTwissButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>

       <!-- Algorithm -->
       <item row="1" column="3">
        <widget class="QLabel" name="labelAlgorithm">
         <property name="text">
          <string>Reconstruction:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QComboBox" name="algorithmComboBox">
         <item>
          <property name="text">
           <string>2D projected (x, y)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4D with coupling</string>
          </property>
         </item>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="middleHorizontalLayout">

      <item>
       <widget class="QGroupBox" name="groupBoxControls">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>1000</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="controlsLayout">

         <item>
          <widget class="QLabel" name="labelScreensUsed">
           <property name="text">
            <string>OTRs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="screensListWidget">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>

         <item>
          <spacer name="controlsSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>

      <!-- RIGHT: PLOTS -->
      <item>
       <widget class="QGroupBox" name="groupBoxPlots">
        <property name="title">
         <string>Plots</string>
        </property>
        <layout class="QGridLayout" name="gridLayoutPlots">
         <item row="0" column="0">
          <widget class="QFrame" name="plotFrame11">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="plotFrame12">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="plotFrame21">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="plotFrame22">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>

     </layout>
    </item>


    <!-- BOTTOM BUTTONS -->
    <item>
     <layout class="QHBoxLayout" name="bottomButtonsLayout">
      <item>
       <spacer name="bottomSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="measureButton">
        <property name="text">
         <string>Measure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportButton">
        <property name="text">
         <string>Save as...</string>
        </property>
       </widget>
      </item>

     </layout>
    </item>

   </layout>
  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>

 </widget>
 <resources/>
 <connections/>
</ui>
